src/style.css

```diff
@@ -93,6 +104,299 @@ header {
+/* ------------------------
+recent work section style
+---------------------------- */
+
+.section-works {
+  margin-bottom: 120px;
+}
+
+.works-title {
+  height: 83px;
+  margin-top: 120px;
+  margin-bottom: 24px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: var(--color-black);
+  border-bottom: 1px solid #212121;
+}
+
+.feature-image {
+  margin-top: 80px;
+}
+
+.title-post {
+  margin-top: 24px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #091e42;
+}
+
+.post-description {
+  height: 124px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.button {
+  display: flex;
+  margin-top: 12px 0;
+  gap: 12px;
+  width: 236px;
+  height: 40px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: var(--color-white);
+}
+
+.button-list {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 12px 75px 22px 16px;
+  gap: 12px;
+  width: 236px;
+  height: 40px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: var(--color-white);
+}
+
+.button-list button {
+  display: flex;
+  align-items: center;
+  padding: 10px 12px;
+  height: 40px;
+  background-color: rgba(255, 255, 255, 0.24);
+  border: 1px solid rgba(255, 255, 255, 0.24);
+}
+
+.about-me button {
+  margin-top: 24px;
+  margin-right: 175px;
+  padding: 12px;
+  width: 161px;
+  height: 48px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  background-color: var(--color-Safety-Orange);
+  color: var(--color-white);
+}
+
+.card button {
+  color: var(--color-white);
+}
+
+.button button:hover {
+  color: #ff6b00;
+  background: #fff5e1;
+  border: 1px solid var(--color-Safety-Orange);
+}
+
+.button button:active {
+  color: #f4f1f0;
+}
+
+.Recent-work:nth-child(1) .button button {
+  padding: 10px 12px;
+  background-color: var(--color-white);
+  border: 1px solid #8993a4;
+}
+
+.Recent-work .see-project-btn,
+.card .see-project-btn {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: var(--color-white);
+}
+
+.Recent-work:nth-child(1) .button {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+}
+
+.Recent-work .see-project-btn {
+  display: flex;
+  align-items: center;
+  margin-top: 12px;
+  margin-bottom: 24px;
+  border: 1px solid var(--color-Safety-Orange);
+  padding: 12px;
+  width: 124px;
+  height: 48px;
+  background-color: var(--color-Safety-Orange);
+}
```

Comment:
- Just as the previous CR mentioned, your `See Project button` is currently not matching the Figma design. In the Figma design, the button doesn't break into two lines. However, it does in yours and per the project requirements you're to stick to the Figma design as closely as possible. So I suggest you fix this before you resubmit for review :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/213520522-edb339f4-d2c7-4d93-8415-5422be4df070.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/213520657-892fc18d-7283-4e8f-80a1-2b83a5c39dc4.png)